corehq/apps/enterprise/enterprise.py

```diff
@@ -209,6 +211,8 @@ def total_for_domain(self, domain_obj):
 
 class EnterpriseFormReport(EnterpriseReport):
     title = _('Mobile Form Submissions')
+    MAXIMUM_USERS_PER_DOMAIN = getattr(settings, 'ENTERPRISE_REPORT_DOMAIN_USER_LIMIT', 20000)
+    MAXIMUM_ROWS_PER_REQUEST = getattr(settings, 'ENTERPRISE_REPORT_ROW_LIMIT', 1000000)
```

Comment:
Not worth committing, but just as an FYI in case you were unaware, you can use `_` as a separator to make integer literals easier to read:
```suggestion
    MAXIMUM_USERS_PER_DOMAIN = getattr(settings, 'ENTERPRISE_REPORT_DOMAIN_USER_LIMIT', 20_000)
    MAXIMUM_ROWS_PER_REQUEST = getattr(settings, 'ENTERPRISE_REPORT_ROW_LIMIT', 1_000_000)
```

Reply:
Thanks, that's really handy!